scripts/service_migration.py

```diff
@@ -0,0 +1,87 @@
+import copy
+from kubernetes import client, config
+from kubernetes.client.exceptions import ApiException
+import argparse
+import os.path
+from pprint import pprint
+
+def update_annot(v1, svc):    
+    '''
+    ************************************************************************
+    Take dictionary of original service, copies Service.status.load_balancer.ingress[0].ip to Service.metadata.annotations and patches Service
+    --------------
+    Arguments:
+    v1 :: CoreV1Api object
+    svc :: original service as dictionary
+    --------------
+    Returns:
+    None
+    ************************************************************************
+    '''
+    
+    svc['metadata']['annotations']['load-balancer-ips'] = svc['status']['load_balancer']['ingress'][0]['ip']
```

Comment:
make the annotation a constant defined in script

Reply:
constant added to script